vhost-user-backend/src/handler.rs

```diff
@@ -30,7 +30,7 @@ use super::event_loop::{VringEpollError, VringEpollResult};
 use super::vring::VringT;
 use super::GM;
 
-const MAX_MEM_SLOTS: u64 = 32;
+const MAX_MEM_SLOTS: u64 = 509;
```

Comment:
The changes make sense to me, and being aligned with `vhost` kernel module as well.
How about adding a comment here to shortly explain why we use this value?